docs/Data loading, Migration/Import CSV data.md

```diff
@@ -1,104 +1,87 @@
+If you are using `mariadb` client, the [`--local-infile` option](https://mariadb.com/docs/skysql-dbaas/ref/mdb/cli/mariadb/local-infile/) must be specified.
 
-If you are using `mariadb` client, the [`--local-infile` option](https://mariadb.com/docs/skysql-dbaas/ref/mdb/cli/mariadb/local-infile/) must be specified.
-
-### **Enable Local Infiles in SkySQL**
+### Enable Local Infiles in SkySQL
 
 Support for local infiles must be enabled on the SkySQL service.
 
-For SkySQL services that use MariaDB Enterprise Server and MariaDB Enterprise ColumnStore, the [local_infile system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/) must be enabled:
+For SkySQL services that use MariaDB Enterprise Server and MariaDB Enterprise ColumnStore, the [`local_infile` system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/) must be enabled:
 
-- For Replicated Transactions and Single Node Transactions services, the [local_infile system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/) is `OFF` by default
+- For Replicated Transactions and Single Node Transactions services, the [`local_infile` system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/) is `OFF` by default.
 
-[Configuration Manager](https://mariadb.com/docs/skysql-dbaas/service-management/nr-configuration-management/) can be used to modify the value of the [local_infile system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/).
+[Configuration Manager](https://mariadb.com/docs/skysql-dbaas/service-management/nr-configuration-management/) can be used to modify the value of the [`local_infile` system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/).
 
-## **Import Data**
+## Import Data
 
-1. Determine the [connection parameters](https://mariadb.com/docs/skysql-dbaas/connect/nr-client-connections/) for your MariaDB SkySQL service.
-2. Connect with the `mariadb` client and specify the [-local-infile option](https://mariadb.com/docs/skysql-dbaas/ref/mdb/cli/mariadb/local-infile/), which is needed by the next step:
+1. Determine the [connection parameters](https://mariadb.com/docs/skysql-dbaas/connect/nr-client-connections/) for your MariaDB SkySQL service.
+2. Connect with the `mariadb` client and specify the [`--local-infile` option](https://mariadb.com/docs/skysql-dbaas/ref/mdb/cli/mariadb/local-infile/), which is needed by the next step:
 
-```bash
-mariadb --host FULLY_QUALIFIED_DOMAIN_NAME --port TCP_PORT \
-      --user DATABASE_USER --password \
-      --ssl-verify-server-cert \
-      --ssl-ca ~/PATH_TO_PEM_FILE \
-      --default-character-set=utf8 \
-      --local-infile
-```
+    ```bash
```

Comment:
We don't require or even accept a PEM file anymore for connections. Check the 'how to connect' section.